src/main/java/use_cases/PostManager.java

```diff
@@ -0,0 +1,73 @@
+package use_cases;
+
+import entities.*;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+/*
+store all the posts existing in the app,
+add a new post, comment/like a Post
+ */
+public class PostManager {
+    private ArrayList<Post> posts;
+
+    public PostManager(){
+        this.posts = new ArrayList<Post>();
```

Comment:
Could we get the Posts from the DatabaseManager? You can merge the main branch into this branch using the commands I showed you earlier, to get the new changes to the DatabaseManager